packages/form-js-editor/src/features/properties-panel/entries/InputKeyValuesSourceEntry.js

```diff
@@ -15,7 +15,7 @@ export default function InputKeyValuesSourceEntry(props) {
 
   const description = <div>
     Define which input property to populate the values from.
-    <br /><br />The input property must follow this schema:
+    <br /><br />The input property may be an array of primitives or alternatively follow this schema:
```

Comment:
Can we expect our users to know what "primitives" mean? Should we maybe show this via example as well? This could blow up the description, though.

Reply:
Exactly, that's why I opted against it. This is explained in more detail in the docs anyways. I'll rename it to `array of simple values`.